### DISCUSSION TITLE
Prismic Example 404 when deployed to Vercel

### DISCUSSION THREAD

**Discussion by thedevdavid** [vercel/next.js#23965] [Help]
# Bug report

When you deploy prismic example to Vercel, it shows 404 for dynamic routes. But it works on local dev or local build. There are no errors at build.

## To Reproduce

1. Follow README https://github.com/vercel/next.js/tree/canary/examples/cms-prismic
2. Try to open a dynamic route on the deployed version (ex. https://cms-prismic-app-43ji45pn4.vercel.app/posts/test-post)

## Additional context

Tried it with [Next on Netlify](https://github.com/netlify/next-on-netlify) too, it's the same.

**Comment by kongweiying2** [comment]
I'm having this issue as well.

Here is the result when I use the "Deploy" button on the repo, and use Vercel to deploy from a template:

The home page works:
https://elenta-landing-7abx644h5-stultus-studios.vercel.app/
But the blog posts 404:
https://elenta-landing-7abx644h5-stultus-studios.vercel.app/posts/jane-austen

It's the same result when I use `yarn create next-app --example cms-prismic cms-prismic-app` and then push to GitHub, and then use Vercel to deploy:

https://elenta-landing-local.vercel.app/

However, I noticed that when I run the build locally, using `yarn dev`, clicking on a blog post title from the homepage, in order to access the blog post does nothing. Maybe this is causing the issue? However, if I control click, or manually paste the URL into the address bar, the blog post is displayed properly, without any 404s.
